nominees/public-utility-data-liberation-project.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Public Utility Data Liberation Project",
+  "aliases": [
+    "PUDL"
+  ],
+  "description": "The Public Utility Data Liberation (PUDL) project takes US energy system information that’s already publicly available, and makes it publicly usable, by cleaning, standardizing, and cross-linking utility data from different sources in a single database.",
+  "website": "http://catalyst.coop/pudl",
+  "license": [
+    {
+      "spdx": "CC-BY-4.0",
+      "licenseURL": "https://github.com/catalyst-cooperative/pudl"
+    },
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/catalyst-cooperative/pudl/blob/main/LICENSE.txt"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 7,
+      "evidenceText": "Substantially increasing the share of renewable energy in the global energy mix and promoting investment in energy infrastructure will require transparency in resource planning decisions and in the models that underpin energy system data. Transparency in resource planning and open energy system models must be built on open and accessible energy system data.",
+      "evidenceURL": "https://datacollaboratives.org/cases/the-public-utility-data-liberation-pudl-project.html"
+    }
+  ],
+  "sectors": [
+    "Energy"
+  ],
+  "type": [
+    "software",
+    "data"
+  ],
+  "repositories": [
+    {
+      "name": "Public Utility Data Liberation Project",
+      "url": "https://github.com/catalyst-cooperative/pudl"
+    }
```

Comment:
Not sure whether the [data portal](https://data.catalyst.coop/) should be included as a repository.

Reply:
Resolved in commit 5298ad0